### ISSUE TITLE
Extra whitespace being added to classes when parsed

### ISSUE THREAD

**Issue by dl-web-dev** [schoero/eslint-plugin-better-tailwindcss#303]
### Description

I Have noticed that when using the enforce consistent line wrapping rule, classes get parsed with extra space as seen in the screenshot.

<img width="522" height="34" alt="Image" src="https://github.com/user-attachments/assets/494bf7cf-9b41-4b42-9dee-9577d424f8fe" />

Is there any way to prevent this from happening ?

### Flavor

TSX

### Code Input

```tsx
<div
          className={`
            grid grid-cols-1 gap-4
            md:grid-cols-2
            lg:grid-cols-3
            xl:grid-cols-4
          `}
        >

        </div>
```

### Expected Behavior

```tsx
"grid grid-cols-1 gap-4 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4"
```

### Actual Behavior

```tsx
" grid grid-cols-1 gap-4 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 "
```

### ESLint Log

```shell
N/A
```

### ESLint Config

```typescript
'better-tailwindcss': {
        callees: [
          'cn',
          'clsxm',
          [
            'cva',
            [
              {
                match: 'strings'
              },
              {
                match: 'objectValues',
                pathPattern: '^variants\\.[^.]+\\.[^.]+$'
              },
              {
                match: 'objectValues',
                pathPattern: '^compoundVariants\\[\\d+\\]\\.(?:className|class)$'
              }
            ]
          ]
        ],
        entryPoint: 'src/styles/globals.css',
        printWidth: 150
```

### Versions

- ESLint: 
- Parser: 
- Plugin: 
- Node:

**Comment by schoero** [comment]
I'm going to close this as this is not something I can fix in this plugin.
